packages/field-highlighter/test/visual/lumo/field-highlighter.test.js

```diff
@@ -0,0 +1,210 @@
+import '../../../theme/lumo/vaadin-field-highlighter.js';
+import { setUsers } from '../common.js';
+
+describe('field-highlighter', () => {
+  let div, element;
+
+  beforeEach(() => {
+    div = document.createElement('div');
+    div.style.display = 'inline-block';
+    div.style.padding = '20px';
+    div.style.height = '150px';
+  });
+
+  describe('checkbox', () => {
+    beforeEach(async () => {
+      element = fixtureSync(`<vaadin-checkbox label="I agree with terms"></vaadin-checkbox>`, div);
+      setUsers(element);
+      await nextFrame();
+    });
+
+    it('default', async () => {
+      await visualDiff(div, 'checkbox');
+    });
+
+    it('focused', async () => {
+      await sendKeys({ press: 'Tab' });
+      await nextFrame();
+      await visualDiff(div, 'checkbox-focused');
+    });
+  });
+
+  describe('checkbox-group', () => {
+    beforeEach(async () => {
+      element = fixtureSync(
+        `
+          <vaadin-checkbox-group>
+            <vaadin-checkbox name="user" value="1" label="Option 1"></vaadin-checkbox>
+            <vaadin-checkbox name="user" value="2" label="Option 2"></vaadin-checkbox>
+            <vaadin-checkbox name="user" value="3" label="Option 3"></vaadin-checkbox>
+            <vaadin-checkbox name="user" value="4" label="Option 4"></vaadin-checkbox>
+          </vaadin-checkbox-group>
+        `,
+        div
+      );
+      setUsers(element);
+      await nextFrame();
+    });
+
+    it('default', async () => {
+      await visualDiff(div, 'checkbox-group');
+    });
+
+    it('focused', async () => {
+      await sendKeys({ press: 'Tab' });
+      await nextFrame();
+      await visualDiff(div, 'checkbox-group-focused');
+    });
+  });
+
+  describe('date-time-picker', () => {
+    beforeEach(async () => {
+      element = fixtureSync(`<vaadin-date-time-picker></vaadin-date-time-picker>`, div);
+      setUsers(element);
+      await nextFrame();
+    });
+
+    it('default', async () => {
+      await visualDiff(div, 'date-time-picker');
+    });
+
+    it('focused', async () => {
+      await sendKeys({ press: 'Tab' });
+      await nextFrame();
+      await visualDiff(div, 'date-time-picker-focused');
+    });
+  });
+
+  describe('list-box', () => {
+    beforeEach(async () => {
+      div.style.height = '250px';
+      element = fixtureSync(
+        `
+          <vaadin-list-box>
+            <vaadin-item>Option one</vaadin-item>
+            <vaadin-item>Option two</vaadin-item>
+            <vaadin-item>Option three</vaadin-item>
+            <vaadin-item>Option four</vaadin-item>
+          </vaadin-list-box>
+        `,
+        div
+      );
+      await nextFrame();
+      setUsers(element);
+      await nextFrame();
+    });
+
+    it('default', async () => {
+      await visualDiff(div, 'list-box');
+    });
+
+    it('focused', async () => {
+      await sendKeys({ press: 'Tab' });
+      await sendKeys({ press: 'Space' });
```

Comment:
What is the purpose for the additional `Space` press, wouldn't the `Tab` press be enough to get an item focused?

Reply:
The `Space` press is there for selecting an item so it would have a checkmark. I think we can keep it.